packages/server/src/schema/uuid/page/resolvers.ts

```diff
@@ -148,4 +153,21 @@ export const resolvers: TypeResolvers<Page> &
       return { success: true, query: {} }
     },
   },
+  PageQuery: {
+    async pages(_parent, payload, { dataSources }) {
+      const { pages } = await dataSources.model.serlo.getPages({
+        instance: payload.input.instance,
```

Comment:
```suggestion
        instance: payload.input?.instance,
```
Needed since we will make `PagesInput` optional